src/features/common/Layout/Navbar/microComponents/NavbarDropdownItem.tsx

```diff
@@ -0,0 +1,47 @@
+import type { JSX } from 'react';
+
+import { useTranslations } from 'next-intl';
+import styles from '../NavbarDropdown.module.scss';
+import { PlatformIcon } from '../../../../../../public/assets/images/icons/NavbarDropdownIcons';
+
+type TitleTranslationKeys = 'platform';
+
+type NavbarDropdownItemProps = {
+  headerKey: keyof typeof navbarDropdownIcons;
+  description: 'platformDescription';
+  title: TitleTranslationKeys;
+  visible: boolean;
+  link: string;
+};
+
+const navbarDropdownIcons: Record<TitleTranslationKeys, JSX.Element> = {
+  platform: <PlatformIcon />,
+};
+
+const NavbarDropdownItem = ({
+  headerKey,
```

Comment:
Please call this `SingleNavItem` as we discussed.

Reply:
Since it belongs to the navbar mega menu section, I have updated it to NavbarMenuItem instead of calling it SingleNavItem.